### PR TITLE
 Add Persian language to dontasktoask

### PR DESCRIPTION
**Pull request by usefss**
this is the Persian translation of don't ask to ask in kind of slang way not very official

### CODE REVIEW

**Comment by usefss** [comment]
its been 7 days... :sleeping: should I close this?

File: fa/index.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    blockquote p {
+      display: inline;
+    }
+
+    @media screen and (min-width: 400px) {
+      blockquote > .name {
+        padding-right: 8px;
+      }
+    }
+  </style>
```
Comment (tulir) [inline fa/index.html:78]:
```suggestion
  <link rel="stylesheet" type="text/css" href="/style.css">
  <style>
    * {
      text-align: right;
    }
    blockquote {
      border-left: none !important;
      border-right: 10px solid #CCC;
    }
    @media screen and (min-width: 400px) {
      blockquote {
        display: block !important;
      }
      blockquote > * {
        display: block !important;
      }
    }
  </style>
```

File: fa/index.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="ar">
```
Comment (tulir) [inline fa/index.html:2]:
```suggestion
<html lang="fa">
```

File: fa/index.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+<head>
+  <title>برای پرسیدن سوال نپرس، فقط بپرسش</title>
+  <meta property="og:title" content="برای پرسیدن سوال نپرس، فقط بپرسش" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com/ar" />
```
Comment (tulir) [inline fa/index.html:7]:
```suggestion
  <meta property="og:url" content="https://dontasktoask.com/fa" />
```

File: fa/index.html [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+  <meta property="og:locale" content="ar_TN" />
```
Comment (tulir) [inline fa/index.html:8]: Not sure if this is the correct country, correct if not

```suggestion
  <meta property="og:locale" content="fa_IR" />
```

File: fa/index.html [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+      Further reading: <a href="https://stackoverflow.com/help/how-to-ask">How do I ask a good question?</a>،
+      or if you have more time: <a href="http://catb.org/~esr/faqs/smart-questions.html">How To Ask Questions The Smart Way</a>.
+    </p>
+  </main>
+  <footer>
+    Content mostly stolen from <a href="http://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
+    -
+    Translated by painor <a href="https://github.com/painor">@painor</a>
+    -
+    <a href="https://github.com/maunium/dontasktoask.com">Source on Github</a>
```
Comment (tulir) [inline fa/index.html:139]: This part should probably be translated too (or at least change the "Translated by" name and link)

**Comment by usefss** [comment]
ok changes have been committed I guess you will add styles.css later... :thinking:

**Pull request merged**